csrc/mma_type.h

```diff
@@ -171,7 +171,10 @@ class TORCH_CUDA_CU_API MmaBuilder {
   //! TODO: This step will very likely be removed in a follow up PR. All of
   //!  the options configured here could actually be inferred from fusion IR
   //!  once we are feature complete.
+  //! TODO: Remove variant that accepts TensorView, variant with MmaOp
```

Comment:
I haven't removed unused variant of the function as it will introduce a number of similar changes in the tests. I plan to have it done in the follow up PR.